### PR TITLE
Support FieldValue.increment

### PR DESCRIPTION
**Pull request by A-Babin**
Completes #225 

I also took the liberty of removing the default `timestampsInSnapshots` setting, since [it's no longer required](https://firebase.google.com/support/release-notes/js#cloud-firestore_16) and will log an error.

### CODE REVIEW

File: lib/src/interop/firestore_interop.dart [hunk 1]
```diff
@@ -321,7 +323,16 @@ abstract class Settings {
   external bool get timestampsInSnapshots;
   external set timestampsInSnapshots(bool v);
 
-  external factory Settings({bool timestampsInSnapshots});
```
Comment (kevmoo) [inline lib/src/interop/firestore_interop.dart]: revert all of this stuff. Unrelated!

**Comment by kevmoo** [comment]
Thanks @A-Babin !

**Comment by A-Babin** [comment]
Glad to help! I've got support for storage `list` and `listAll` basically ready too, submitting soon.

**Comment by A-Babin** [comment]
Glad to help! I've got support for storage `list` and `listAll` basically ready too, submitting soon.

**Pull request closed without merging**